Lambdas/GHOAuth/OAuthLambda.swift

```diff
@@ -0,0 +1,101 @@
+import AsyncHTTPClient
+import AWSLambdaRuntime
+import AWSLambdaEvents
+import DiscordBM
+import Foundation
+import SotoSecretsManager
+import GHHooksLambda
+import Models
+
+@main
+struct GHOAuthHandler: LambdaHandler {
+    typealias Event = APIGatewayV2Request
+    typealias Output = APIGatewayV2Response
+
+    let client: HTTPClient
+    let logger: Logger
+    let secretsRetriever: SecretsRetriever
+
+    init(context: LambdaInitializationContext) {
+        self.client = HTTPClient(eventLoopGroupProvider: .shared(context.eventLoop))
+        self.logger = context.logger
+        let awsClient = AWSClient(httpClientProvider: .shared(client))
+        self.secretsRetriever = SecretsRetriever(awsClient: awsClient, logger: logger)
+    }
+
+    func handle(
+        _ event: APIGatewayV2Request,
+        context: LambdaContext
+    ) async throws -> APIGatewayV2Response {
```

Comment:
This function should not be declared as `throws`.
Technically this _should be_ fine, but in practice, it won't be fine. If this function throws, AWS will just show a unhelpful 5xx to users.
Remove `throws`, and always return a response with a proper error description.

Reply:
We should probably redirect users back to our Discord server, but not sure how. For now you can just return a "we linked your account, get back to the discord server".

Reply:
outdated